theme.json

```diff
@@ -28914,6 +28914,13232 @@
+                },
+                "type": "size",
+                "subType": "padding",
+                "label": "Right"
+              },
+              {
+                "name": "agendaPaddingBottom",
+                "default": "0px",
+                "columns": 6,
+                "styles": [
+                  {
+                    "parentSelector": "[data-widget-package='com.fliplet.dynamic-lists'] [data-dynamic-lists-layout='agenda']",
+                    "selectors": ".agenda-list-holder",
+                    "properties": ["padding-bottom"]
+                  }
+                ],
+                "breakpoints": {
+                  "tablet": {
+                    "name": "agendaPaddingBottomTablet",
+                    "default": "inherit-mobile"
+                  },
+                  "desktop": {
+                    "name": "agendaPaddingBottomDesktop",
+                    "default": "inherit-tablet"
+                  }
+                },
+                "type": "size",
+                "subType": "padding",
+                "label": "Bottom"
+              },
+              {
+                "name": "agendaPaddingLeft",
+                "default": "0px",
+                "columns": 6,
+                "styles": [
+                  {
+                    "parentSelector": "[data-widget-package='com.fliplet.dynamic-lists'] [data-dynamic-lists-layout='agenda']",
+                    "selectors": ".agenda-list-holder",
+                    "properties": ["padding-left"]
+                  }
+                ],
+                "breakpoints": {
+                  "tablet": {
+                    "name": "agendaPaddingLeftTablet",
+                    "default": "inherit-mobile"
+                  },
+                  "desktop": {
+                    "name": "agendaPaddingLeftDesktop",
+                    "default": "inherit-tablet"
+                  }
+                },
+                "type": "size",
+                "subType": "padding",
+                "label": "Left"
+              }
+            ]
+          },
+          {
+            "description": "List item title",
+            "fields": [
+              {
+                "name": "agendaTitleFontFamily",
+                "default": "$bodyFontFamily",
+                "styles": [
+                  {
+                    "parentSelector": "[data-widget-package='com.fliplet.dynamic-lists'] [data-dynamic-lists-layout='agenda']",
+                    "selectors": ".new-agenda-list-container .agenda-cards-wrapper .agenda-list-card-holder .agenda-list-item .agenda-item-content-holder .agenda-item-title",
+                    "properties": ["font-family"]
+                  }
+                ],
+                "breakpoints": {
+                  "tablet": {
+                    "name": "agendaTitleFontFamilyTablet",
+                    "default": "inherit-mobile"
+                  },
+                  "desktop": {
+                    "name": "ag
```

Comment:
Same as before.

Reply:
Not sure why it highlighted the wrong line, but this is because of the package name having the encoded `:`.

Reply:
Fixed